converters/heightfield2shading.cpp

```diff
@@ -251,19 +270,65 @@ struct ImageMapReflectance{
 };
 
 
+
 struct IdColor{
   Color operator()( const unsigned int & aValue ) const{
     return DGtal::Color(aValue);
   }
 };
 
 
+void
+HSVtoRGB
```

Comment:
I think we already have this kind of hsv 2 rgb in the Colormap classes of DGtal. Could you please check ?

Reply:
yes we have but is private in class ColorBrightnessColorMap, perhaps moving it in Color class?

Reply:
Agree, moving these functions to the Color class would be better

Reply:
Could you please create an issue on dgtal for the hsv in Color ?

Reply:
yes but direct in PR: https://github.com/DGtal-team/DGtal/pull/1593

Reply:
And just changing the Dgtal change, sorry for the time shift ;)